cluster-provision/gocli/cmd/run.go

```diff
@@ -652,6 +665,20 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 			}
 		}
 
+		var vmArgsSharedDisks []string
+		if len(sharedDisks) > 0 {
+			for i, size := range sharedDisks {
+				pciOffset := i + 1
+				resource.MustParse(size)
+				disk := fmt.Sprintf("/shared/disk%d.img", i)
+				blockDev := fmt.Sprintf("-blockdev file,filename=%s,node-name=shared-disk-%d,read-only=off,cache.direct=on,cache.no-flush=off", disk, i)
+				device1 := fmt.Sprintf("-device pcie-root-port,id=pci.%d,bus=pcie.0", pciOffset)
+				device2 := fmt.Sprintf("-device virtio-blk-pci,bus=pci.%d,drive=shared-disk-%d,id=shared-disk-%d,share-rw=on,write-cache=on,werror=stop,rerror=stop", pciOffset, i, i)
```

Comment:
@alicefr can you validate these settings?

Reply:
@mhenriks I would disable the cache, `write-cache=off`

Reply:
Hi @alicefr yeah I was a little confused about that setting. I actually took the settings from a kubevirt `shareable` disk. Was surprised to see `write-cache=on`. But then I checked qemu man page [1] and I got the impressions, which may be wrong. Correct me if so!

1. `write-cache` is synonymous with `cache.writeback` on device
2. `none` cache is `cache.writeback=on`, `cache.direct=on`, `cache.no-flush=off`

[1] https://www.qemu.org/docs/master/system/qemu-manpage.html

Reply:
sorry, for the noise, I think you are right, at least from the QEMU doc. Thehn, just ignore my comment